### DISCUSSION TITLE
What's the easiest way to exclude a dataset from being draggable?

### DISCUSSION THREAD

**Discussion by karloluis** [chrispahm/chartjs-plugin-dragdata#136] [Q&A] [answered]
As the question says. There are some datasets that should be draggable and some not. I can think on ways to use the OnDrag events to prevent certain dataset indexes from being updated. 
Is there some property that can configure it this rather than adding custom ways to filter the events from modifying the value?

**Comment by artus9033** [comment, marked as answer]
We should have this documented at a later stage (we are missing solid API docs), so this is a good question. If you take a look at how this is currently implemented, it looks as follows:
- setting a `dragData` key (full path: `data.datasets[index].dragData`) to `false` on an entire dataset
- setting a `dragData` key to `false` on a individual scales (full path: `config.options.scales[index].dragData`)
- setting a `dragData` key to `false` on individual samples (full path: `data.datasets[index].data[elementIndex].dragData`)
